sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 5.2.0-beta.1 (Unreleased)
 
 ### Features Added
+- Added property `ExtractSummaryActions` to `TextAnalyticsActions` to support the new 'extractive text summarization' API. This action can be used to extract relevant sentences from an input document.
```

Comment:
I think we need to be careful about describing this action because it applies a well-known NLP analysis and "relevant sentences" does not sound like the best description. Here is one suggestion:
```suggestion
- Added property `ExtractSummaryActions` to `TextAnalyticsActions` to support the new 'extractive text summarization' API. This action can be used to get a summary for the input document by extracting the most relevant sentences.
```